README.md

```diff
@@ -22,36 +15,28 @@ Latest:
 1. [Run unit tests](#running-unit-tests)
 1. [Caveats](#caveats)
 
-## Dependencies
-
-Both *ListView* and *VirtualListView* need [jquery](http://jquery.com/),
-[handlebars](http://handlebarsjs.com), [ember.js](http://emberjs.com).
-
-*VirtualListView* need an additional dependency: [zynga scroller](https://github.com/zynga/scroller).
-
+## Installation
+  **TODO** - Need to publish addon.
+  * `ember install ember-collection`
 ## Demo
-
-Please, take a look at our live [demo](http://emberjs.com/list-view) and [jsbin](http://emberjs.jsbin.com/) links:
-[first](http://emberjs.jsbin.com/betiyuna/1) and [second](http://emberjs.jsbin.com/fuqob/1).
+**TODO** - Create twiddles with examples. Needs the addon to be published first?
 
 ## Submitting bugs
 
-Please, attach code samples or links to [jsbin](http://emberjs.jsbin.com/) or [jsfiddle](http://jsfiddle.net/).
-It would help us greatly to help you and to improve ember list view.
+**TODO** - Create a base ember twiddle with *EmberCollection* setup.
 
-## Installation
-
-Install *ListView* with EmberCLI using this command.
-
-```bash
-ember install ember-list-view
-```
+It would help us greatly to help you and to improve ember collection.
 
 ## Usage
 
 First, let's create a template:
 ```handlebars
-{{#ember-collection items=model height=500 rowHeight=50 width=500}}
+{{#ember-collection
+  items=model
+  height=500
+  width=800
+  cell-layout=(fixed-grid-layout 800 50) as |item index|
```

Comment:
Hmm... I guess that could work, just seems kinda funky. There are already going to be a bunch of changes from migrating from `list-view` -> `ember-collection`. Could we perhaps do an `ember-watson` command that will do a mechanical upgrade for people?

I just worry about carrying over old syntax cause it kinda of ties our hands, like if  in the future we were able to default to a multi-pass layout. I'll defer to @mmun or @krisselden .